indica/datatypes.py

```diff
@@ -100,6 +100,7 @@
     "impurity": ("Impurity", "none"),
     "beamlet": ("Beamlet", "none"),
     "residence_time": (r"Impurity residence time $\tau$", "time"),
+    "magnetic_field": ("Magnetic field", "magnetic_field"),
```

Comment:
Is this the total magnetic field? I just noticed I've implemented toroidal, poloidal, radial and vertical, but not the total. 
I'd recommend keeping the standard used for the other entries, i.e.:
 "total_magnetic_field": ("$B_{tot}$", "magnetic_field")